src/reference/01-General-Info/05-Using-Sonatype.md

```diff
@@ -206,46 +212,59 @@ Close the staging repository and promote the release to central, by hitting
 
 > *Note:* sbt-sonatype is a third-party plugin meaning it is not covered by Lightbend subscription.
 
-To simplify the usage of the Sonatype's Nexus, add the following line to 
+To simplify the usage of the Sonatype's Nexus, add the following line to
 `project/plugins.sbt` to import the [sbt-sonatype plugin][sbt-sonatype] to your project:
 
 ```
-addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.9.5")
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.9.13")
```

Comment:
Update plugin version to get the fix from: 
PR: https://github.com/ShiftLeftSecurity/sbt-ci-release-early/pull/12
Issue: https://github.com/xerial/sbt-sonatype/issues/214